homeassistant/components/foscam/camera.py

```diff
@@ -1,11 +1,22 @@
 """This component provides basic support for Foscam IP cameras."""
 import logging
+from asyncio import sleep
```

Comment:
Please import `asyncio`. That makes it clear what sleep we're using below in the code.